src/test/java/racingcar/domain/CarsTest.java

```diff
@@ -57,10 +58,18 @@ public void drive_winners(String name1, String name2) {
         carList.add(car1);
         carList.add(car2);
 
-        Cars cars = new Cars(carList);
+        Cars cars = Cars.from(carList);
         car1.drive(5);
         car2.drive(4);
 
         assertThat(cars.getWinners().size()).isEqualTo(2);
     }
```

Comment:
테스트하고자하는 목표에 집중하면 좋을 것 같아요. 

여기서는 우승자를 파악하기만 하면 되죠. 5, 4라는 숫자가 들어감으로 인하여 읽는 사람이 추가적인 컨텍스트를 파악해야할거 같아요. 

```suggestion
        Cars cars = Cars.from(carList);
        
        Int drivableNumber = 4;
        
        car1.drive(drivableNumber);
        car2.drive(drivableNumber);

        assertThat(cars.getWinners().size()).isEqualTo(2);
    }
```

예시로 작성한 코드라 문법적으로 틀릴 수도 있지만 요런 식으로 맥락을 줄여서 사용자가 읽기 편하게 만들어볼 수 있겠다 싶네요~!